translator/translate/otel/processor/metricstransformprocessor/translator_test.go

```diff
@@ -46,12 +72,21 @@ func TestTranslator(t *testing.T) {
 			if err == nil {
 				require.NotNil(t, got)
 				gotCfg, ok := got.(*metricstransformprocessor.Config)
-				require.True(t, ok)
-				require.Equal(t, tc.want["transforms"].(map[string]interface{})["include"], gotCfg.Transforms[0].MetricIncludeFilter.Include)
-				require.Equal(t, tc.want["transforms"].(map[string]interface{})["match_type"], fmt.Sprint(gotCfg.Transforms[0].MetricIncludeFilter.MatchType))
-				require.Equal(t, tc.want["transforms"].(map[string]interface{})["experimental_match_labels"], gotCfg.Transforms[0].MetricIncludeFilter.MatchLabels)
-				require.Equal(t, tc.want["transforms"].(map[string]interface{})["action"], fmt.Sprint(gotCfg.Transforms[0].Action))
-				require.Equal(t, tc.want["transforms"].(map[string]interface{})["new_name"], gotCfg.Transforms[0].NewName)
+				if name != "JMXMetricsSection" {
```

Comment:
Would prefer that we had a more generic way of validating the result.

Reply:
could add this as part of input struct